v20.2/create-role.md

```diff
@@ -1,45 +1,52 @@
 {{site.data.alerts.end}}
 
+For more information, see the [Authorization](authorization.html) documentation on roles and users.
+
 ## Considerations
 
-- Role names:
-    - Are case-insensitive
-    - Must start with either a letter or underscore
-    - Must contain only letters, numbers, periods, or underscores
-    - Must be between 1 and 63 characters.
-- After creating roles, you must [grant them privileges to databases and tables](grant.html).
-- Roles and users can be members of roles.
-- Roles and users share the same namespace and must be unique.
-- All privileges of a role are inherited by all of its members.
-- There is no limit to the number of members in a role.
-- Roles cannot log in. They do not have a password and cannot use certificates.
-- Membership loops are not allowed (direct: `A is a member of B is a member of A` or indirect: `A is a member of B is a member of C ... is a member of A`).
+{% include {{ page.version.version }}/sql/role-considerations.md %}
 
 ## Required privileges
 
- To create other roles, the role must be a member of the `admin` role or have the [`CREATEROLE`](#create-a-role-that-can-create-other-roles-and-manage-authentication-methods-for-the-new-roles) parameter set.
+Unless a role is a member of the admin role, additional [privileges](#parameters) are required to manage other roles.
+
+- To create other roles, a role must have the [`CREATEROLE`](#create-a-role-that-can-create-other-roles-and-manage-authentication-methods-for-the-new-roles) parameter set.
+- To add the `LOGIN` capability for other roles so that they may log in as users, a role must also have the [`CREATELOGIN`](#create-a-role-that-can-create-other-roles-and-manage-authentication-methods-for-the-new-roles) parameter set. 
+- To be able to grant or revoke membership to a role for additional roles, a member of the role must be set as a [role admin](authorization.html#role-admin) for that role. 
 
 ## Synopsis
 
 <section>{% include {{ page.version.version }}/sql/diagrams/create_role.html %}</section>
 
 ## Parameters
 
+<style>
+table td:first-child {
+    min-width: 225px;
+}
+</style>
+
 | Parameter | Description |
 ------------|--------------
-`name` | The name of the role you want to create. Role names are case-insensitive; must start with either a letter or underscore; must contain only letters, numbers, periods, or underscores; and must be between 1 and 63 characters.<br><br>Note that roles and [users](create-user.html) share the same namespace and must be unique.
+`name` | The name of the role you want to create. See the [Considerations](#considerations) section for important naming guidelines.
 `CREATELOGIN`/`NOCREATELOGIN` | Allow or disallow the role to manage authentication using the `WITH PASSWORD`, `VALID UNTIL`, and `LOGIN/NOLOGIN` parameters. <br><br>By default, the parameter is set to `NOCREATELOGIN` for all non-admin roles.
```

Comment:
The list of possible role options should be part of the authorization page, and referred to here. I think it's confusing to explain this in the "create role" page when really it's an authorization matter and users can inherit these options via role membership.

Reply:
I see your point. I am logging this in the follow-up issue.